tests/Etc/tenant-schema.dump

```diff
@@ -0,0 +1,66 @@
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
```

Comment:
This file is used in the `migrate_command_loads_schema_state` test to check if it actually used the schema to migrate the tenants.  I contains a _schema_users_ table that is not included in the default test migrations.

The actual schema dump generated during the test will have the name of "tenant-schema-test.dump". I will gitignore it so that someone cannot commit it by mistake during local testing